build/components/render/web-design-system.html

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
I'm confused about the name of this file? Is this kinda like the main index.html?